peps/pep-0772.rst

```diff
@@ -0,0 +1,455 @@
+explicitly abstain. Members with conflicts of interest on a particular vote
+must abstain. Passing requires support from a majority of non-abstaining
+council members.
+
+Whenever possible, the council is expected to share its decisions and processes
+publicly in a timely fashion.
+
+Delegations
+-----------
+
+The Python Steering Council would be expected to delegate decision making to
+the Packaging Council for PEPs related to the Python packaging.  The bodies
+would work together on issues that intersect the packaging domain and language
+stewardship (including the CPython implementation, standard library, and
+distribution).
+
+The PSF Board would be encouraged to formally deprecate the Packaging Workgroup
+and the Packaging Council would wholly take on the responsibilities of the
+PSF's Packaging Workgroup.
+
+The PyPA would be expected to work with the Packaging Council to establish a
+decision making process that governs the technical projects under the PyPA
+umbrella.
+
+Processes
+=========
+
+Election of the council
+-----------------------
+
+A council election consists of two phases:
+
+* Phase 1: A member of the Packaging community (defined later in this document)
+  can self-nominate themselves for the council elections.
+* Phase 2: Each core team member can assign zero to five stars to each
+  candidate. Voting is performed anonymously. The outcome of the vote is
+  determined using the `STAR voting system`_, modified to use the `Multi-winner
+  Bloc STAR`_ approach. If a tie occurs, it may be resolved by mutual agreement
+  among the candidates, or else the winner will be chosen at random.
+
+Each phase should last two weeks.
+
+The election process is managed by a returns officer nominated by the outgoing
+Packaging Council. For the initial election, the returns officer will be
+nominated by the PSF Executive Director.
+
+The PSF shall maintain records of the elections as well as run the annual
+election for the council.
+
+Term
+----
+
+There shall be two cohorts of council members: Cohort A composed of two (2)
+members and Cohort B composed of three (3) members.
+
+Each council member shall be elected for a two-year term, unless they are
+replacing a council member that resigned, was removed, or otherwise becomes
+vacant, in which case such replacement members shall be elected to a term
+sufficient to complete a two-year term as measured from the term of the
+original cohort.
+
+Each council member's term will be two (2) years, from when their elections are
+finalised until the next elections for their cohort are finalised.
+
+Only for elections involving the entire council (like the initial council
+election), the two candidates receiving the highest number of votes shall be
+designated Cohort A with a two (2) year term, and the three candidates
+receiving the highest number of votes after shall be designated Cohort B with a
+one (1) year term.
+
+There are no term limits for individual council members.
```

Comment:
As a proceduralist myself, I hesistate to say that this seems overly formalistic -- experience with the steering council shows a lot of natural overlap. If you want to stagger the terms, perhaps just say:

```
For the first election of the Council, and any subsequent elections where
all five positions are vacant, the two highest-ranking candidates shall be
elected for a two-year term, and the next three highest-ranking candidates
shall be elected for a one-year term.
```

Reply:
Is this suggestion to replace all the text? Because it's missing the wording around the two year term stuff.

FWIW, this text is modelled after the PSF's bylaws, so... yea, it's definitely very formal. 😅

Reply:
@AA-Turner the only scenario I can think of where all five positions are vacant _except_ for the first vote would be a mass resignation of the PC.  I think the original text covers that; are you thinking of something else?

FWIW, the two-cohort idea has been floated for the SC, to ensure continuity, but a) it fortunately hasn't come up in practice yet; b) would require a PEP 13 change process to achieve.  This is one of those sections that deliberately deviate from PEP 13 rhyming, I think for a good reason.

Reply:
Re-reading the first comment and @warsaw's comment today... I feel like this is a suggestion for changing about what's being proposed, rather than something editorial about the way things are written.

If my understanding is correct, I suggest we defer further discussion on this until the PEP is up on discuss.python.org (to avoid non-editorial discussions in this PR).